scripts/calculate_trending_scores_daily.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+
+import _init_path  # noqa: F401 Imported for its side effect of setting PYTHONPATH
+
+from openlibrary.config import load_config
+from openlibrary.plugins.worksearch.code import execute_solr_query
+from openlibrary.plugins.worksearch.search import get_solr
+from scripts.solr_builder.solr_builder.fn_to_cli import FnToCLI
+
+
+def fetch_works(current_day: int):
+    resp = execute_solr_query(
```

Comment:
Note: This currently doesn't do any retries, which means if we have a sporadic/temporary network issue, this will result in trending scores being off for seven days. Note it won't zero things out or anything (phew!) but the values will just be off and use yesterday's scores effectively.

Chatted with @benbdeitch , and we think this is ok to move forward with now for testing, but we should add a comment here noting the risk!